### DISCUSSION TITLE
Can you cut a new release of Brotli4j?

### DISCUSSION THREAD

**Discussion by joakime** [hyperxpro/Brotli4j#148] [General]
I'd like to start using the `com.aayushatharva.brotli4j.encoder.EncoderJNI.Wrapper` from Eclipse Jetty.

These were recently made available via the changes due to ...

* #144 
* #145

**Comment by hyperxpro** [comment]
```xml
<dependency>
    <groupId>com.aayushatharva.brotli4j</groupId>
    <artifactId>brotli4j</artifactId>
    <version>1.17.0</version>
</dependency>
```

There you go!
